internal/iostreams/survey.go

```diff
@@ -491,6 +495,9 @@ func (io *IOStreams) retrieveFlagValue(flagset []*pflag.Flag) (*pflag.Flag, erro
 		return nil, nil
 	}
 	for _, opt := range flagset {
+		if opt == nil {
+			continue
+		}
```

Comment:
👁️‍🗨️ Here we avoid checking nonexistent flags for a changed value which is useful since the `create` command doesn't provide a "language" flag.

Reply:
Interesting how the survey is kinda getting in the way. Thanks for improving it!